dispatchr-ruby/db/schema.rb

```diff
@@ -34,8 +34,8 @@
 
   create_table "items", force: :cascade do |t|
     t.string   "name"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", default: '2016-11-13 15:36:04', null: false
+    t.datetime "updated_at", default: '2016-11-13 15:36:04', null: false
```

Comment:
Should default to an arbitrary date. Let rails default it to the current date for you.